src/aws-cpp-sdk-core/source/utils/logging/DefaultCRTLogSystem.cpp

```diff
@@ -22,57 +22,36 @@ namespace Aws
     {
         namespace Logging
         {
-            DefaultCRTLogSystem::DefaultCRTLogSystem(LogLevel logLevel) :
-                m_logLevel(logLevel),
-                m_stopLogging(false),
-                m_logsProcessed(0)
+            DefaultCRTLogSystem::DefaultCRTLogSystem(LogLevel logLevel)
             {
+                AWS_UNREFERENCED_PARAM(logLevel); // will use one from the main SDK logger
             }
 
             DefaultCRTLogSystem::~DefaultCRTLogSystem()
             {
-                m_stopLogging = true;
```

Comment:
np: couldnt we just move this out to the header as `~DefaultCRTLogSystem() = default` then?